server/server.js

```diff
@@ -4,8 +4,11 @@ import app from "./app";
 import { connectDb, disconnectDb } from "./db";
 import config from "./utils/config";
 import logger from "./utils/logger";
-
+import { Logger, LogOutput } from "concurrently";
```

Comment:
I think this line made it in accidentally. Remove it please

Reply:
done